features/logout/impl/src/main/kotlin/io/element/android/features/logout/impl/LogoutPresenter.kt

```diff
@@ -47,8 +52,15 @@ class LogoutPresenter @Inject constructor(
             mutableStateOf(Async.Uninitialized)
         }
 
-        val backupUploadState: BackupUploadState by remember {
-            encryptionService.waitForBackupUploadSteadyState()
+        val secureStorageFlag by featureFlagService.isFeatureEnabledFlow(FeatureFlags.SecureStorage)
```

Comment:
Is setting to false by default no enough?

Reply:
If we set to false, it's `false` then `true`, which is not ideal, both for test (more intermediate states) and for prod.
I am looking how to improve this with `Async`.

Reply:
Actually, I start with a `null` value, and it seems to be OK (still testing it)